### PR TITLE
Add types for vuex-i18n

### PR DESCRIPTION
**Pull request by jaeggerr**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
@jaeggerr Please fix the failures indicated in the Travis CI log.

**Comment by jaeggerr** [comment]
> Dependency vuex not in whitelist; please make a pull request to types-publisher adding it.

I'm not sure of what I should do.
vuex is a dependency for the library and the test file, but I do not need it to declare the definitions.
Can I remove the dependency in package.json ?

**Comment by andy-ms** [comment]
@jaeggerr I've added it to the whitelist.

File: types/vuex-i18n/index.d.ts
Comment (plantain-00) [inline types/vuex-i18n/index.d.ts]: Can be changed to `// TypeScript Version: 2.3`

**Review by plantain-00 — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by third party. PR appears ready to merge pending review by a maintainer.

**Pull request merged**
